microprot/snakemake/Snakefile

```diff
@@ -3,192 +3,276 @@
 # "qsub -k eo -m n -l nodes=1:ppn={cluster.n} -l mem={cluster.mem}gb \
 # -l walltime={cluster.time}" --directory "$@"
 
-import os
-import skbio
+# snakemake -s Snakefile_minimal -n
+
 import yaml
 import re
-from microprot.scripts import split_sequence, mask_sequence, calculate_Nf
+import sys
+from glob import glob
+from os import makedirs
+from os.path import basename, splitext, exists
+from shutil import copyfile
+from skbio import io
+
+sys.path.append('/projects/microprot')
```

Comment:
I think it makes sense to keep the dir structure as is, but give the user the opportunity to split inputs (for which read permissions are sufficient) and all kind of results (for which also write permissions are required). Furthermore, you could have a different output dir than me, but we both operate on the same input.
Can you have a MICROPROT_OUTPUT which is set to MICROPROT_INPUT by default?

Regarding special characters: I suggest to do a replacement of the chars in the name: `name.replace(['/', '\', '|'], ['_','_','_'])`